Makefile

```diff
@@ -93,153 +90,163 @@ else
 
 .PHONY: stack-logs-sipi
 stack-logs-sipi: ## prints out and follows the logs of the 'sipi' container running in knora-stack.
-	docker-compose -f docker/knora.docker-compose.yml logs -f sipi
+	@docker-compose -f docker-compose.yml logs -f sipi
 
 .PHONY: stack-logs-sipi-no-follow
 stack-logs-sipi-no-follow: ## prints out the logs of the 'sipi' container running in knora-stack.
-	docker-compose -f docker/knora.docker-compose.yml logs sipi
+	@docker-compose -f docker-compose.yml logs sipi
 
 .PHONY: stack-logs-redis
 stack-logs-redis: ## prints out and follows the logs of the 'redis' container running in knora-stack.
-	docker-compose -f docker/knora.docker-compose.yml logs -f redis
+	@docker-compose -f docker-compose.yml logs -f redis
 
 .PHONY: stack-logs-api
 stack-logs-api: ## prints out and follows the logs of the 'api' container running in knora-stack.
-	docker-compose -f docker/knora.docker-compose.yml logs -f api
+	@docker-compose -f docker-compose.yml logs -f api
 
 .PHONY: stack-logs-api-no-follow
 stack-logs-api-no-follow: ## prints out the logs of the 'api' container running in knora-stack.
-	docker-compose -f docker/knora.docker-compose.yml logs api
+	docker-compose -f docker-compose.yml logs api
 
 .PHONY: stack-logs-salsah1
 stack-logs-salsah1: ## prints out and follows the logs of the 'salsah1' container running in knora-stack.
-	docker-compose -f docker/knora.docker-compose.yml logs -f salsah1
+	docker-compose -f docker-compose.yml logs -f salsah1
 
 .PHONY: stack-health
 stack-health:
 	curl -f 0.0.0.0:3333/health
 
 .PHONY: stack-status
 stack-status:
-	docker-compose -f docker/knora.docker-compose.yml ps
+	docker-compose -f docker-compose.yml ps
 
 .PHONY: stack-down
 stack-down: ## stops the knora-stack.
-	docker-compose -f docker/knora.docker-compose.yml down
+	docker-compose -f docker-compose.yml down
 
 .PHONY: stack-down-delete-volumes
 stack-down-delete-volumes: ## stops the knora-stack and delete any created volumes (deletes the database!).
-	docker-compose -f docker/knora.docker-compose.yml down --volumes
+	docker-compose -f docker-compose.yml down --volumes
 
 .PHONY: stack-config
 stack-config: env-file
-	docker-compose -f docker/knora.docker-compose.yml config
+	docker-compose -f docker-compose.yml config
 
 ## stack without api
 .PHONY: stack-without-api
 stack-without-api: stack-up ## starts the knora-stack without knora-api: graphdb, sipi, redis, salsah1.
-	docker-compose -f docker/knora.docker-compose.yml stop api
+	@docker-compose -f docker-compose.yml stop api
 
-## stack without api and sipi
 .PHONY: stack-without-api-and-sipi
 stack-without-api-and-sipi: stack-up ## starts the knora-stack without knora-api and sipi: graphdb, redis, salsah1.
-	docker-compose -f docker/knora.docker-compose.yml stop api
-	docker-compose -f docker/knora.docker-compose.yml stop sipi
+	@docker-compose -f docker-compose.yml stop api
+	@docker-compose -f docker-compose.yml stop sipi
 
-.PHONY: test-only
```

Comment:
Here you have removed the `test-only`, what is supposed to be used instead?

Reply:
you can run the following:

`bazel test //webapi/src/test/scala/org/knora/webapi/e2e/v1:SearchV1R2RSpec`

Reply:
You can call single spec files. You can also create filegroups, that group certain spec files in a meaningful way. The final goal is that you should be able to always call `bazel test //...` which will only run the tests to which the input has changed.

Reply:
Is there a way to run a single test within a spec? Currently I do this in SBT like this:

```
webapi/testOnly *OntologyV2R2RSpec -- -z select1
```

Where `select1` is something unique in the string description of the test. This is described here:

https://www.scalatest.org/user_guide/using_the_runner#selectingSuitesAndTests

Reply:
Ah, cool. I was thinking about this the order day. Bazel provides a way of passing stuff to the underlying test runner. I will try it out and report my findings.

Reply:
ok, found it:

`bazel test //webapi/src/test/scala/org/knora/webapi/e2e/v1:SearchV1R2RSpec --test_arg=-z --test_arg="gaga"`

where `gaga` is the universal unique string :-) This will only run the tests inside the spec which description contains `gaga`.

Reply:
Cool, can you please document it so I can copy and paste that whenever I need it? :)